lib/facter/util/ciscoos.rb

```diff
@@ -0,0 +1,22 @@
+module Facter
+  module Util
+    module CiscoOS 
+
+      # @file follows the same format as os-release
+      # @see http://www.freedesktop.org/software/systemd/man/os-release.html
+      def self.cisco_release(file)
```

Comment:
@peterhuene is there a standard for this I should point them to? I'll note also that the release file for NXOS is multiline - here's an example:

ID=nexus
ID_LIKE=wrlinux
NAME=Nexus
HOME_URL=http://www.cisco.com
BUILD_ID=TBD
CISCO_RELEASE_INFO=/etc/os-release